modules/tinymce/src/core/main/ts/keyboard/PageUpDownKeys.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import Editor from '../api/Editor';
+import VK from '../api/util/VK';
+import * as InlineBoundariesNavigation from './InlineBoundariesNavigation';
+import * as MatchKeys from './MatchKeys';
+
+const executeKeydownOverride = (editor: Editor, evt: KeyboardEvent) => {
```

Comment:
This should be named executeKeyupAction or something since it's not really overriding and it's on keyup not keydown.

Reply:
updated